README.md

```diff
@@ -11,7 +11,11 @@ Phantom is an app which is used to track bus movements.
 
 ### Starting the application
 
+<<<<<<< HEAD
 - Run `npm start` To start your application on `localhost:3000`.
 ### Testing
 
 - Use `npm test` to run the application's test.
+=======
+- Run `npm run dev` To start your application.
+>>>>>>> chore(react):setup React
```

Comment:
same here removes thes unwanted text of conflicts resolving

Reply:
worked on that